unittests/test_sanity_functions.py

```diff
@@ -650,6 +653,46 @@ def test_extractall(self):
         for expected, v in enumerate(res, start=1):
             assert expected == v
 
+    def test_extractall_multiple_tags(self):
+        # Check multiple numeric groups
+        res = sn.evaluate(sn.extractall(
+            r'Number: (\d+) (\d+)', self.tempfile, (1, 2)))
+        for expected, v in enumerate(res, start=1):
+            assert str(expected) == v[0]
+            assert str(2 * expected) == v[1]
```

Comment:
You can be more compact here (same later on):

```suggestion
            assert str(2*expected) == v[1]
```